lib/combinat.gi

```diff
@@ -2446,6 +2446,72 @@ InstallGlobalFunction( "IteratorOfPartitions", function( n )
     end );
 
 
+#############################################################################
+##
+#F  IteratorOfPartitionsSet( <set> )
+##
+##  all unordered partitions of the set <A>set</A> into pairwise disjoint
+##  nonempty sets.
+##
+InstallGlobalFunction( IteratorOfPartitionsSet, function( s )
+    local nextIterator, isDone, m, out, i, shallowCopy;
+
+      if not IsSet(s) then
+          Error( "<s> must be a set" );
+      fi;
```

Comment:
```suggestion
    if not IsSet(s) then
      Error( "<s> must be a set" );
    fi;
```
To match the style of the rest of the code

Reply:
I have fixed the spacing